### ISSUE TITLE
[Bug] Idk whats happening when changing window focus

### ISSUE THREAD

**Issue by im-izz** [glzr-io/glazewm#751]
### Describe the bug

Its just not working and doing weird taskbar flashes when using the shortcuts (**doesn't happen in the cli version** but its not convenient solution since I'm going to have that cmd window open always)

https://github.com/user-attachments/assets/5eb80bf0-3520-4ede-a3b7-acd794114498

### Version number

3.2.0

**Comment by im-izz** [comment]
Okay so the reason why the cli version works while the gui version not is because that the cli doesn't have admin privileges and after changing the gui compatibility setting for running as admin to off and running without admin privileges make it work fine but why tho? and why would admin privileges be needed in window management? and if its needed how do I fix the problem with this all

**Comment by im-izz** [comment]
Oh alright, I fixed it already by adding this script (.ps1) to my task scheduler:
```
$exePath = "C:\Program Files\glzr.io\GlazeWM\glazewm.exe"

Start-Process -FilePath $exePath -Verb runAs -WindowStyle Hidden
```
in task scheduler program  is: 'powershell.exe'
and argument is '-ExecutionPolicy Bypass -WindowStyle Hidden -File "C:\Users\Izz\.glzr\start-glazewm.ps1"'
and checking the run with highest privileges option for silent admin privileges.

and for that app is called keyviz :)

**Comment by im-izz** [comment]
Interesting, we will wait for the next update hopefully fixes many problems, btw how did you get that application title in zebar? I'm already customizing it and would love to add that feature to my zebar too.

**Comment by im-izz** [comment]
> the title in zebar has some proble, it only update when change windows focus ,So it's not mentioned in the document, because he's not ready
its alright i just want to display the current program like "explorer.exe" and would be even better if it was just "explorer" without the exntension
@DreamMaoMao

**Comment by lars-berger** [comment]
Added a fix for this in v3.3.0 👍

**Comment by im-izz** [comment]
> Added a fix for this in v3.3.0 👍

Now I can't change focus while having the focus follow mouse option enabled
